examples/chip-tool/commands/common/CHIPCommand.h

```diff
@@ -82,7 +82,6 @@ class CHIPCommand : public Command
     static void RunQueuedCommand(intptr_t commandArg);
 
     CHIP_ERROR mCommandExitStatus = CHIP_ERROR_INTERNAL;
-    chip::Controller::ExampleOperationalCredentialsIssuer mOpCredsIssuer;
```

Comment:
As far as I can tell, this needs to stay....

Reply:
This needs to stay based on refactors from @vivien-apple that were merged after I was done. It caused a merge issue I did not catch right away. It's fixed now.